### DISCUSSION TITLE
How to get the coverage percent by method name?

### DISCUSSION THREAD

**Discussion by apbertoletti** [FortuneN/FineCodeCoverage#332] [Q&A] [answered]
Hi everyone,

I started to test the Fine Code Coverage plugin in Visual Studio and I liked it so much, congratulation on the work.

But just a question: how can I get the coverage percent by method name?

Thanks

**Comment by tonyhallett** [comment, marked as answer]
https://github.com/FortuneN/FineCodeCoverage/issues/412#issuecomment-2033947430
